Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jinaai/jina:latest
+
+# install requirements before copying the workspace
+COPY requirements.txt /requirements.txt
+RUN pip install --default-timeout=1000 --compile -r requirements.txt
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+ENTRYPOINT ["jina", "executor", "--uses", "config.yml"]
```

Comment:
We don't need `Dockerfile` in this case. The hubble will take care of it in this simple case.

Reply:
Addressed, please see [here](https://github.com/jina-ai/executor-simpleindexer/pull/1/commits/6f12e89a4194f5cb85751ba0deca9f7443d97274) for the change.